submissions/vakulinina/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;600;700&display=swap" rel="stylesheet">
+  <title>Book your seat</title>
+</head>
+
+<body>
+  <main>
+    <header>
+      <h1>Buy tickets</h1>
+    </header>
+    <article class="movie">
+      <div class="movie-poster"></div>
+      <div class="movie-description">
+        <h2>Alita: Battle Angel</h2>
+        <p>2019 &bull; Action | Sci-Fi</p>
+        <p><b>Synopsys</b></p>
```

Comment:
Why you use tag `b` in this situation?

Reply:
Well, to make it bold? )) As it's kind of sub-header. Should it be `h3` or something?

Reply:
H3 with styles on bold will be good one